### PR TITLE
Adding Option type on LinkedList struct

### PR DESCRIPTION
**Pull request by gabrielgusn**
Making the Option type addition on the Linked List struct as told on the 1 UTC session

### CODE REVIEW

**Comment by delcin-raj** [comment]
Could you add examples on how to create LinkedList?
Like empty list, singleton list and list with 2 elements?

**Comment by gabrielgusn** [comment]
Yes, I will do it!

File: curriculum/module_3/3_linked_list.md [hunk 1]
```diff
@@ -72,7 +72,72 @@ This one is pretty simple,
...
+        value: 24,
+        next: None,
+    };
+    // Create the first node with the value 42 and next pointing to the second node
+    let first_node = Node {
+        value: 42,
+        next: Some(Box::new(second_node)),
+    };
+
+    // Create a singleton linked list by setting head to the node
```
Comment (delcin-raj) [inline curriculum/module_3/3_linked_list.md]: This example is for list with 2 nodes right?

**Review by delcin-raj — CHANGES_REQUESTED** [review]
small correction

File: curriculum/module_3/3_linked_list.md (hunk 1 above)
Comment (gabrielgusn) [reply curriculum/module_3/3_linked_list.md]: Yes

Comment (gabrielgusn) [reply curriculum/module_3/3_linked_list.md]: I will change the comment

**Review by delcin-raj — APPROVED** [review]
Thank you for contributing!

**Comment by gabrielgusn** [comment]
Thanks!

**Pull request merged**